### ISSUE TITLE
Acces user phone number 

### ISSUE THREAD

**Issue by Mateeen** [mirzemehdi/KMPAuth#79]
can we access user phone number form google singin? how to do that

**Comment by mirzemehdi** [comment]
I don't think it is available from google sign in side, since phone number is considered sensitive info. But if you find any way to make it possible, or some source please reopen the issue
